scio-core/src/main/scala/com/spotify/scio/io/JdbcOptions.scala

```diff
@@ -0,0 +1,49 @@
+package com.spotify.scio.io
+
+import org.apache.beam.sdk.io.jdbc.JdbcIO.{PreparedStatementSetter, RowMapper, StatementPreparator}
+
+
+object JdbcOptions {
+
+  /**
+    * Options require to create a connection with remote database.
+    * @param username - database username
+    * @param password - database password
+    * @param connectionUrl - connection url i.e "jdbc:mysql://[host]:[port]/db?"
+    * @param driverClassName - database driver class name
+    * @return database connection options
+    */
+  case class DbConnectionOptions(username: String,
+                                password: String,
```

Comment:
@labianchin this is database level password which only use to access the database, we can use google cloud SDK credential to access the Cloud SQL instance in my example code I used google mysql SocketFactory to create this secure socket `i.e. cloudSqlInstance=%s&socketFactory=com.google.cloud.sql.mysql.SocketFactory` which is secure as SSL and we don't need to whitelist any IP.